feature/about/src/main/java/io/github/droidkaigi/confsched2020/about/ui/AboutFragment.kt

```diff
@@ -101,15 +103,20 @@ class AboutFragment : Fragment(R.layout.fragment_about), Injectable {
                 aboutItemFactory.create(
                     getString(R.string.about_item_licence)
                 ) {
-                    // TODO go licence-page
+                    OssLicensesMenuActivity.setActivityTitle(
+                        this.getString(R.string.licenses_label)
+                    )
+                    Navigation.findNavController(
```

Comment:
Can we use `findNavController()`? 👀

Reply:
Thanks! `findNavController()` can be usable.